students/ScotchWSplenda/lesson01/assignment/inventory_management/main.py

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env python3
+# pylint: disable=R0913,R0903,W0603
+'''launches the user interface for the inventory management system'''
+
+import sys
+import market_prices as mp
+import inventory_class as ic
+import furniture_class as fc
+import electric_appliances_class as eac
+
+
+def main_menu(user_prompt=None):
+    '''some stuff12'''
+    valid_prompts = {"1": add_new_item,
+                     "2": item_info,
+                     "q": exit_program}
+    options = list(valid_prompts.keys())
+
+    while user_prompt not in valid_prompts:
+        options_str = ("{}" + (", {}") * (len(options)-1)).format(*options)
+        print(f"please choose from the following options ({options_str}):")
+        print("1. add a new item to the inventory")
+        print("2. get item information")
+        print("q. quit")
+        user_prompt = input(">")
+    return valid_prompts.get(user_prompt)
+
+
+def get_price(item_code):
+    '''some stuff13'''
+    return(f"get price for {item_code}")
```

Comment:
This works, but you don't need parenthesis in a return statement, so it could just be:

`return f"get price for {item_code}"`